plugins/mumble_plugin_win32.h

```diff
@@ -105,16 +154,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
+#ifndef USE_PLUGIN_LEGACY_PTR
+	static VOID *pModule;
+#endif
 	pModule=getModuleAddr(modname ? modname : procname);
 	if (!pModule) {
 		dwPid = 0;
 		return false;
 	}
+	pModule32 = reinterpret_cast<uintptr32_t>(pModule);
```

Comment:
Hmm..

Well, that case could be handled by adding a `bool is64bit` argument to `initialize()` instead.